app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,23 @@
+class NotificationsController < ApplicationController
+  before_action :set_room
+
+  def index
+    respond_to do |format|
+      format.json { render json: @notifications }
+    end
+  end
+
+  def read
+    @notifications.update_all(read_at: Time.now)
```

Comment:
never use `Time.now`, always `Time.current`. Maybe time to add `rubocop` so it'd enforce some basic things?